docs/section10/10-14.md

```diff
@@ -0,0 +1,159 @@
+            - SYSTEM TYPE 3 and 3a  (>=225,000)  
+            - Else, capacity is greater than or equal to 225,000 Btuh: `Else:`  
+                - Set the boolean variable for zone aggregation factor needed to FALSE: `zone_aggregation_factor_undefined_and_needed = FALSE`                   
+                - Loop through each of the efficiency_metric_types associated with the heating system until COMBUSTION_EFFICIENCY is found. Get the associated value of the efficiency from the list of efficiency_metric_values associated with the heating_system_b : `For x in range(len(heating_system_b.efficiency_metric_types)):`  
+                    - Check if the efficiency metric equals COMBUSTION_EFFICIENCY: `if heating_system_b.efficiency_metric_types[x] == "COMBUSTION_EFFICIENCY":`  
+                        - Set the modeled efficiency value: `modeled_efficiency_b = heating_system_b.efficiency_metric_values[x]`  
+                        - Break out of loop if efficiency is found: `break`    
+                - Set expected_baseline_eff_b equal to 80% Ec: `expected_baseline_eff_b = 0.8`  
+        - SYSTEM 4
+        - Else, system type is 4:`Else:` 
+            - Get dictionary list of baseline zones and the associated HVAC systems: `hvac_zone_list_w_area_dict_b = get_hvac_zone_list_w_area (B_RMR)`   
+            - Get list of zones that the HVAC system serves (this is to get the zone associated with sys 3 or 4): `zone_list_b = hvac_zone_list_w_area_dict_b[hvac_b.id]["ZONE_LIST"]`   
+            Get the cooling capacity (needed for determining heating efficiency for system 4)  
+            - Check if rated cooling capacity does not equal null, if it does not equal null then set the cooling capacity equal to this data element's value: `if cooling_system_b.rated_total_cool_capacity != Null:`  
+                - Set cooling capacity equal to the rated cooling capacity: `total_cool_capacity_b = cooling_system_b.rated_total_cool_capacity`  
+            - Else if, check that design cooling capacity does not equal null:`Elif cooling_system_b.design_total_cool_capacity != Null:`   
+                - Set cooling capacity equal to the design cooling capacity: `total_cool_capacity_b = cooling_system_b.design_total_cool_capacity`  
+            - Else, set the was_capacity_not_defined boolean variable equal to TRUE: `Else: was_cool_capacity_not_defined = TRUE`   
+          
+            - SYSTEM 4 <65000   
+            - Check if cooling capacity is <65,000 Btuh OR the zone aggregation factor was defined and the capacity is less than 65000 Btuh: `if (zone_list_b[0].aggregation_factor != Null AND total_heating_capacity_b/zone_list_b[0].aggregation_factor < 65000) OR total_cooling_capacity_b < 65000 AND was_cool_capacity_not_defined == FALSE:`   
+                - Check if the capacity is greater than or equal to 65000 and zone aggregation not defined: `if total_cooling_capacity_b >= 65000 AND zone_list_b[0].aggregation_factor == Null :`   
```

Comment:
@claperle Does this need to be moved into the SYSTEM 4  >= 135,000 Btu/h block instead?

Reply:
I modified to make the zone_aggregation_factor_undefined_and_needed always TRUE for this condition. Reading through it I think it is all good across the other conditions as well. No need to make adjustments to those. Let me know if I am missing something.